repr.go

```diff
@@ -193,6 +195,11 @@ func (p *Printer) reprValue(seen map[reflect.Value]bool, v reflect.Value, indent
 	}
 	t := v.Type()
 
+	if t == rawJSONType {
+		fmt.Fprintf(p.w, "json.RawMessage(`%s`)", v.Bytes())
```

Comment:
Unfortunately this is not going to suffice - if the string contains a backtick, this will output invalid Go code.